web/libs/datamanager/src/components/Filters/FilterLine/FilterLine.jsx

```diff
@@ -68,15 +67,15 @@ export const FilterLine = observer(({ filter, availableFilters, index, view, sid
       </GroupWrapper>
       <Elem name="remove">
         <Button
-          look="danger"
-          size="small"
-          style={{ border: "none" }}
+          look="string"
+          size="smaller"
           onClick={(e) => {
             e.stopPropagation();
             filter.delete();
           }}
-          icon={<Icon icon={IconClose} size={12} />}
-        />
+        >
+          <IconTrash className="!h-4 !w-4" />
```

Comment:
You use `size` in all other icons, so let's change it for consistency?
```suggestion
          <IconTrash size={12} />
```